source/shared/net/mobile-wallet/export-import-mw.rst

```diff
@@ -1,20 +1,17 @@
 .. _export-import-mw:
 
-=============================================
-Export or import your identities and accounts
-=============================================
+=============================================================
+Make a backup of identities and accounts in the Mobile Wallet
+=============================================================
 
 .. contents::
    :local:
    :backlinks: none
 
-We recommend that you backup your identities and accounts by exporting them from the Mobile Wallet. This way,
-all your accounts, identities, and your address book are saved in a file you can store in a safe location.
+To make sure that you have a backup of your accounts, identities, and addresses, we strongly recommend that you export the data from the Mobile Wallet to a file. This way, all your accounts, identities, and your address book are saved in a file you can store in a safe location.
```

Comment:
I'd suggest to make it a bit clearer here as well. we recommend that you export the data and then move the export file to a location off your phone. I don't think the last sentence is strong enough to cover it.